sdcm/utils/data_validator.py

```diff
@@ -430,43 +470,48 @@ def validate_range_expected_to_change(self, session, during_nemesis=False):
             if not during_nemesis:
                 LOGGER.debug(f'Verify updated row. View {views_set[0]}')
             if not views_set[3]:
-                LOGGER.error(f"Can't start validation for {views_set[0]}. "
-                             f"Copying expected data failed. See error above")
+                DataValidatorEvent(type='error', name='UpdatedRowsValidator', status=Severity.ERROR,
+                                   error=f"Can't start validation for {views_set[0]}. "
+                                         f"Copying expected data failed. See error above")
                 return
 
             before_update_rows = self.longevity_self_object.fetch_all_rows(
                 session=session, default_fetch_size=self.DEFAULT_FETCH_SIZE,
                 statement=f"SELECT {partition_keys} FROM {views_set[0]}",
                 verbose=not during_nemesis)
             if not before_update_rows:
-                LOGGER.error(f"Can't validate updated rows. "
-                             f"Fetch all rows from {views_set[0]} failed. See error above")
+                DataValidatorEvent(type='error', name='UpdatedRowsValidator', status=Severity.ERROR,
+                                   error=f"Can't validate updated rows. "
+                                         f"Fetch all rows from {views_set[0]} failed. See error above")
```

Comment:
while looking on the event, will i see any error event above?

Reply:
maybe: "see details in the sct.log"?

Reply:
> while looking on the event, will i see any error event above?

yes, it will be error from fetch_all_rows function (tester.py). The error will be printed by retrying decorator

Reply:
> maybe: "see details in the sct.log"?

I mean that the error should be searched in the log above from this line. Changed to ```See error above in the sct.log```